client/src/components/addName.js

```diff
@@ -38,6 +38,14 @@ function AddName() {
     return value.charAt(0).toUpperCase() + value.slice(1);
   };
 
+  const handleKeyPress = (event) => {
+    if (event.keyCode === 13) {
+      // console.log('The enter key was pressed');
+      setValue('name', capitalize(inputValue));
```

Comment:
An action like this where we capitalize the name before sending it to the API could be seen as a cleanup step that occurs before submit. There could probably be a wrapper function that calls "cleanup" stuff and handle submit. 

That's probably too abstract for this scenario, but just a pattern to think about. I've seen frontends that require heavy data transformation before posting to the API.

Reply:
Thanks for looking over this.  A wrapper function is an interesting approach.  I'll do some homework on that later.